src/libraries/System.Text.Json/tests/Serialization/Value.ReadTests.GenericCollections.cs

```diff
@@ -997,16 +997,6 @@ public static void ReadKeyValuePairOfList()
             Assert.Equal(3, input.Value[2]);
         }
 
-        [Fact]
-        public static void ReadKeyValuePairOfKeyValuePair()
```

Comment:
Was this a dup?  Wondering why it's being deleted.

Reply:
Yes, see the first bullet point of the second set in the description, there's a Theory testing the exact same thing, and the data that this method uses is already part of the `InlineData` of the Theory.